tests/ensure_provider_tests.py

```diff
@@ -78,6 +78,9 @@
     },
     "playbooks/tests_802_1x_updated.yml": {},
     "playbooks/tests_802_1x.yml": {},
+    "playbooks/tests_eth_pci_address_match.yml": {
+        EXTRA_RUN_CONDITION: "ansible_distribution == 'Fedora' and ansible_distribution_major_version >= '33'",
```

Comment:
Why can we not test this on CentOS/RHEL 8 or 9?

Reply:
`NMSettingMatch:path` was included since NM-1.26.  NM version in CentOS/RHEL 8 is  1.20

Reply:
Are there any plans to update NM to 1.26 in EL8?

Reply:
CentOS Stream 8 has 1.30: http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/

Reply:
Also, the scripts supports defining the minimal NM version: https://github.com/linux-system-roles/network/blob/main/tests/ensure_provider_tests.py#L64

Reply:
the following is the version info that I get from qcow2 image 'CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2' :
```
[root@ibm-p8-kvm-03-guest-02 ~]# rpm -qa | grep NetworkManager
NetworkManager-libnm-1.20.0-3.el8.x86_64
NetworkManager-tui-1.20.0-3.el8.x86_64
NetworkManager-1.20.0-3.el8.x86_64
NetworkManager-team-1.20.0-3.el8.x86_64
```

Reply:
> > > > bus-info in qcow2 image:
> > > > ```
> > > > [root@localhost ~]# sudo lshw -c network -businfo 
> > > > Bus info          Device      Class          Description
> > > > ========================================================
> > > > pci@0000:00:03.0              network        Virtio network device
> > > > virtio@0          eth0        network        Ethernet interface
> > > > ```
> > > 
> > > 
> > > I've tried several different virtualized systems - they all display the information like this. Which is quite different than a bare metal system. Does this mean that our current virtual CI environment is not suitable for testing this feature?
> > 
> > 
> > We never tested against eth0 to avoid breaking the SSH connection to the machine. I guess if we can add another virtual NIC to the machines (is this now possible),
> 
> Not possible with the current standard-inventory-qcow2 script. It might be possible once we move to baseos CI which is based on OpenStack. I suppose trying to do something like this in the test playbook:
> 
> ```
>     - command: ip link add eth1 type veth peer eth1p
>     - command: ip link set up eth1p
>     - command: ip link add eth2 type veth peer eth2p
>     - command: ip link set up eth2p
> ```
> 
> will not register the interface as a PCI device?
> 

tested, I did not get bus-info(path value) for 'veth'  device

Reply:
@richm, I just realized that there is only one CI environment for RHEL 8 (RHEL-8.3). Previously we have RHEL-8.1 and RHEL-8.3(staging). 
For image 'CentOS-8-GenericCloud-8.3.2011-20201204.2.x86_64.qcow2', NM version is 1.26.0